packages/dynamite/dynamite/lib/src/builder/client.dart

```diff
@@ -119,15 +119,14 @@ super(
         for (final tag in tags) {
           final client = clientName(tag.name);
 
-          b.methods.add(
-            Method(
+          b.fields.add(
+            Field(
```

Comment:
Is there a reason they are not marked as final?

I don't think there is a reason we should allow changing the value later on.

Reply:
Nope, just forgot it